app/facades/forecast_facade.rb

```diff
@@ -1,13 +1,16 @@
 class ForecastFacade
-  attr_reader :lat,
-              :lon
+  attr_reader :location
 
-  def initialize(lat, lon)
-    @lat = lat
-    @lon = lon
+  def initialize(location)
+    @lat = lat_lon(location).lat
+    @lon = lat_lon(location).lon
     @service = WeatherService.new
   end
 
+  def lat_lon(location)
```

Comment:
To keep this call out of the controller! Nice work figuring out how to do this so seamlessly